forc-pkg/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "forc-pkg"
+version = "0.6.0"
+authors = ["Fuel Labs <contact@fuel.sh>"]
+edition = "2021"
+homepage = "https://fuel.network/"
+license = "Apache-2.0"
+repository = "https://github.com/FuelLabs/sway"
+description = "Building, locking, fetching and updating sway projects as Forc packages."
+
+[dependencies]
+anyhow = "1"
+forc-util = { version = "0.6", path = "../forc-util" }
+git2 = "0.14"
+petgraph = { version = "0.6", features = ["serde-1"] }
+semver = { version = "1.0", features = ["serde"] }
+serde = { version = "1.0", features = ["derive"] }
+sway-core = { version = "0.6", path = "../sway-core" }
+sway-types = { version = "0.6", path = "../sway-types" }
+sway-utils = { version = "0.6", path = "../sway-utils" }
```

Comment:
Wait no, only make the non-fuel deps major.minor. Fuel deps need to be exact otherwise publishing won't work! 😂